sampleconfig/orderer.yaml

```diff
@@ -66,6 +66,14 @@ General:
     MaxRecvMsgSize: 104857600
     # Max message size in bytes the GRPC server and client can send
     MaxSendMsgSize: 104857600
+    Throttling:
+        # Rate is the maximum rate (transactions per second) for all clients combined.
+        # A zero rate disables throttling.
```

Comment:
Since the orderer.yaml serves as the only documentation for this feature, can you expand a bit. e.g. mention that it is for transaction submission, that it applies at both the client and org level, and mention the behavior when threshold is exceeded.

Reply:
done